lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala

```diff
@@ -153,13 +153,20 @@ object TraceConverter {
   }
 
   private def spanFromStorageSpan(span: Storage.Span, executionGroups: Seq[Domain.ExecutionGroup], spanTimes: SpanTimes): Domain.Span = {
+
+    val tags = span.tags match {
+      case None => Seq()
+      case Some(tags) => tags.map(tag => Domain.Tag(tag.key, tag.vStr))
+    }
```

Comment:
Sounds like
```
val tags =  span.tags.getOrElse(Seq()).map(tag => Domain.Tag(tag.key, tag.vStr)))
````
might be more compact. In fact, do we really need Storage.Span.tags to be Option[Seq]? Can it be just Seq?

Reply:
I think, you are right; Storage.Stan.tags better be Seq

Reply:
I tried to change all Option[Seq] to just Seq and some of my tests failed with null pointer exception. So for now i leave it with options